lib/helpers.js

```diff
@@ -124,18 +124,19 @@ async function runQuery(functionName, queryString, queryParams, pool, clientPara
   }
 }
 
-function getAlertTypeDisplayName(alertType) {
+function getAlertTypeDisplayName(alertType, language, t) {
```

Comment:
The brave-alert-lib repo has quite extensive documentation. Could you please go through the `README.md` and see if there is anything that needs to be added/removed/changed due to the code changes in this PR? Thanks!